src/containers/Careers.js

```diff
@@ -56,60 +56,66 @@ class Careers extends Component {
     this.props.fetchCareers();
   }
 
+  componentWillUnmount() {}
+
+  hideButtons = () => {
+    return this.props.careers.careerCards.length + 1 === this.state.currentCard;
+  };
+
   swingShow = () => {
-    if (
-      this.props.careers.dataFetched &&
-      this.props.careers.careerCards.length
-    ) {
+    if (this.props.careers.dataFetched) {
       return (
-        <Swing
-          tagName="div"
-          setStack={stack => this.setState({ stack: stack })}
-          ref="stack"
-          throwout={e => {
-            this.setState({ currentCard: this.state.currentCard + 1 });
-            this.likeCareer(e);
-          }}
-          config={{
-            allowedDirections: [Direction.LEFT, Direction.RIGHT],
-            maxRotation: 5,
-            minThrowOutDistance: 1000,
-            maxThrowOutDistance: 1400,
-            throwOutConfidence: (xOffset, yOffset, element) => {
-              const xConfidence = Math.min(
-                Math.abs(xOffset) / element.offsetWidth,
-                1
-              );
-              const yConfidence = Math.min(
-                Math.abs(yOffset) / element.offsetHeight,
-                1
-              );
-              if (Math.max(xConfidence, yConfidence) > 0.65) {
-                return 1;
-              }
-              return Math.max(xConfidence, yConfidence);
-            },
-          }}
-        >
-          {this.props.careers.careerCards.map((card, index) => (
-            <CardWrapper
-              ref={`card${index}`}
-              id={`card${card.id}`}
-              key={card.id}
-            >
-              <CareerCard card={card} />
-            </CardWrapper>
-          ))}
-        </Swing>
+        <div>
+          <Swing
+            tagName="div"
+            setStack={stack => this.setState({ stack: stack })}
+            ref="stack"
+            throwout={e => {
+              this.setState({ currentCard: this.state.currentCard + 1 });
+              this.checkThrowRight(e);
+            }}
+            config={{
+              allowedDirections: [Direction.LEFT, Direction.RIGHT],
+              maxRotation: 5,
+              minThrowOutDistance: 1000,
+              maxThrowOutDistance: 1400,
+              throwOutConfidence: (xOffset, yOffset, element) => {
+                const xConfidence = Math.min(
+                  Math.abs(xOffset) / element.offsetWidth,
```

Comment:
What is the `/` doing? :eyes:

Reply:
It's the division symbol. Its some math stuff I took from [here](https://github.com/ssanjun/react-swing/issues/14#issuecomment-323596185) so the cards are easier to throw out